src/MySqlConnector/Serialization/CharacterSet.cs

```diff
@@ -273,4 +273,213 @@ internal enum CharacterSet : ushort
 		Utf8Mb4VietnameseUca900AccentSensitiveCaseSensitive = 300,
 		Utf8Mb4JapaneseUca900AccentSensitiveCaseSensitive = 303,
 	}
+
+	internal static partial class Extensions
+	{
+		public static int MaxLength(this CharacterSet character)
```

Comment:
Can you merge this into the existing `SerializationUtility.GetBytesPerCharacter`?

Reply:
I've pushed a change to `SerializationUtility.GetBytesPerCharacter` so this should be removed in favour of that: 83c1cea6f22ac4547e157a2eb93fccc85e6850a5

Reply:
Sure.